content/blog/eng/angular1-component-a-path-to-angular2.md

```diff
@@ -121,14 +123,14 @@ app.component('bar', {
 
 ```
 
-This would have created a two-way binding with the component’s parent. Whenever the parent would assign a new value, it would be propagated to the child. 
-And vice-versa, if the component assigns a new value it will be copied to the parent, this is two-way data bindings.
+This would have created a **two-way binding** with the component’s parent. Whenever the parent would assign a new value, it would be propagated to the child. 
+And vice-versa, if the component assigns a new value it will be copied to the parent; this is how a two-way data binding operates.
 
 This, while helpful, isn’t a very common scenario in my experience. That’s why Angular has finally introduced the new prospect of one-way bindings.
```

Comment:
Are we sure about this `prospect` word? I'm not sure about the meaning in this context